File: benchmarks/bench_loop.go
// bench_loop.go — Loop + arithmetic benchmark
// Measures: loop overhead, integer arithmetic, variable mutation
package main

import "fmt"

func main() {
	sum := 0
	for i := 0; i < 100000000; i++ {
		sum += i
	}
	fmt.Println(sum)
}

File: benchmarks/bench_loop.rs
// bench_loop.rs — Loop + arithmetic benchmark
// Measures: loop overhead, integer arithmetic, variable mutation

fn main() {
    let mut sum: i64 = 0;
    for i in 0..100_000_000i64 {
        sum += i;
    }
    println!("{}", sum);
}
